src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
@yfarjoun I was getting this idea from the very next sentence which describes a Chimeric alignment as a "set of linear alignments", but I was misinterpreting that. I talked to Chris and I see how supplementary alignments can be on opposite strands (and apparently this is fairly common in some technologies), either from errors or inversion events.